salt/states/win_wusa.py

```diff
@@ -87,15 +88,21 @@ def installed(name, source):
         return ret
 
     # Install the KB
-    __salt__['wusa.install'](cached_source_path)
+
+    additional_comment = ''
```

Comment:
Maybe make this `'.'` by default, so if there's no additional comment, the original output will be produced?

Reply:
I'm not sure I understand.
I think that's the case already, since I "append" additional_comment to the "old" comment. Or maybe you have something else in mind ?

Reply:
You've removed the trailing period from the message a few lines down. When there's no additional comment instead of something like `The sentence.` you get `The sentence` without a period.

If you set this, then when there is no other additional comment, you'll get the original output (i.e. with the period).

Does that make sense?

Reply:
Well I think I understand what you want to say, I agree that there should always be a trailing dot in the comment, but actually I **added** a trailing dot. It's missing in the current version.
With this PR, the trailing dot will always be present, with or without additional comment.

Reply:
😝 doh. You're right. I must have been tired when I was looking at this earlier.